### PR TITLE
Bug 1868850 - Remove failing warn checks from UM

### PR DESCRIPTION
**Pull request by fbertsch**
Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [x] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [x] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [x] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2201)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql [hunk 1]
```diff
@@ -11,9 +11,8 @@
 {{ value_length(column="client_id", expected_length=36, where="submission_date = @submission_date") }}
 
 #warn
-{{ is_unique(columns=["client_id"], where="submission_date = @submission_date") }}
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql:14]: This table should have a grain of 1 entry per client per partition. If this checks fails then there's something going wrong, by just deleting the check we're ignoring the source issue.

File: sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql [hunk 2]
```diff
@@ -11,9 +11,8 @@
...
-
-#warn
+-- The following field is sometimes NULL
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql:14]:
```suggestion
```

File: sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql [hunk 3]
```diff
@@ -11,9 +11,8 @@
...
+-- "normalized_os_version"
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql:15]:
```suggestion
```

File: sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql (hunk 1 above)
Comment (fbertsch) [reply sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql:14]: If that's the case, this should be a `fail` warning and someone should investigate. I do not think we should go into the freeze with a check that's known to fail (and was known to fail historically)

**Comment by fbertsch** [comment]
@ANich context here: https://bugzilla.mozilla.org/show_bug.cgi?id=1775137

tl;dr is we see duplicates in the Fenix tables due to a single client running multiple channels. Until this is fixed upstream this may continue to fail in UM. Until we fix upstream, I believe we should remove this check

**Review by kik-kik — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql (hunk 1 above)
Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql:14]: I think the reason we decided to go with `warn` now is to avoid breaking jobs and progressively switch over to `fail` mode for some checks once we get a feel for how many things fail and do not adhere to our assumptions about the data.

But I agree, this should become a fail check. However, before we do that switch here we should address the duplication inside of the Fenix tables. I'll push a commit to this branch commenting this check out along with a reference to Fenix duplication bug explaining why for now we chose to omit it.

Comment (fbertsch) [reply sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql:14]: That's perfect, it's all merged. Thanks for rounding this out!
